src/lifecycle/render.js

```diff
@@ -14,9 +15,9 @@ export default function (Ctor) {
     if (!elem[shadowRoot]) {
       let sr;
 
-      if (elem.attachShadow) {
+      if (shadowDomV1) {
         sr = elem.attachShadow({ mode: 'open' });
-      } else if (elem.createShadowRoot) {
+      } else if (shadowDomV0) {
         sr = elem.createShadowRoot();
       } else {
         sr = elem;
```

Comment:
Personal style, I guess. I prefer `if else` for small situations. Happy to change if there's a good reason for it.